File: sla/sla.py
num1 = float(input("Digite o primeiro numero: "))
op = input("Digite o operador (+, -, *, /): ")
num2 = float(input("Digite o segundo numero: "))

if op == "+":
    resultado = num1 + num2
elif op == "-":
    resultado = num1 - num2
elif op == "*":
    resultado = num1 * num2
elif op == "/":
    if num2 != 0:
        resultado = num1 / num2
    else:
        print("Erro: divisao por zero!")
        exit()
else:
    print("Operador invalido!")
    exit()

print("Resultado:", resultado)

File: sla/sla.swift
import Foundation

print("Digite o primeiro numero: ", terminator: "")
let num1 = Double(readLine()!)!

print("Digite o operador (+, -, *, /): ", terminator: "")
let op = readLine()!

print("Digite o segundo numero: ", terminator: "")
let num2 = Double(readLine()!)!

var resultado: Double

switch op {
case "+":
    resultado = num1 + num2
case "-":
    resultado = num1 - num2
case "*":
    resultado = num1 * num2
case "/":
    if num2 != 0 {
        resultado = num1 / num2
    } else {
        print("Erro: divisao por zero!")
        exit(1)
    }
default:
    print("Operador invalido!")
    exit(1)
}

print("Resultado: \(resultado)")
